aiomisc/worker_pool/pool.py

```diff
@@ -176,6 +188,21 @@ async def handler() -> None:
             packet_type, identity = await receive()
             assert packet_type == PacketTypes.IDENTITY
 
+            if self.initializer is not None:
+                initializer_done = self.__create_future()
+
+                await step(
+                    self.initializer,
+                    self.initializer_args,
+                    dict(self.initializer_kwargs),
+                    initializer_done
+                )
+
+                try:
+                    await initializer_done
+                except Exception:
+                    log.warning("Initializer function has been failed")
```

Comment:
Maybe re-raise exception here?

Reply:
That fails the pool worker instance, it's more radical then just warning